connector_voicent_helpdesk_ticket/view/queue_job_view.xml

```diff
@@ -0,0 +1,15 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+
+    <record id="view_queue_job_form_inherit_voicent" model="ir.ui.view">
+        <field name="name">queue.job.form.inherit.voicent</field>
+        <field name="model">queue.job</field>
+        <field name="inherit_id" ref="queue_job.view_queue_job_form"/>
```

Comment:
@murtuzasaleh I am pretty sure the `voicent_campaign` field is not in this view.

Reply:
@max3903 It is under the connector_voicent module because of previous scheduler code.
I will remove it from there and add it here.